test/images/porter/README.md

```diff
@@ -1,5 +1,9 @@
 This directory contains go source, Dockerfile and Makefile for making a test
 container which serves requested data on ports specified in ENV variables.
 
+The included localhost.crt is a PEM-encoded TLS cert with SAN IPs
+"127.0.0.1" and "[::1]", expiring at the last second of 2049 (the end
```

Comment:
Everyone is going to get cert errors? no one will see this as being served from 127.0.0.1?